### PR TITLE
Fix #351 Make adding channels easier

### LINKED ISSUE #351: Allow easy adding new channels
Whether there are existing channels or not it would be good to have a button on the interface to add them without going to admin panel. How about the proposition below?

screenshot from 2017-08-09 17-43-28

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
Changelog:
- showing channels bar on homepage for admins even if there are no channels
- added + sign (with WCAG message) on a channels bar to link to channel creation
- Channel's language is a mandatory field now
- Channel's and Noun's language is prefilled with current language
- Channel's order field has been hidden and drag-and-drop interface has been introduced by adminsortable2 library
- New translation: Hint for Channel's is_featured has been added
- WCAG "hide element" classes been added to css

### CODE REVIEW

File: web/main/settings.py [hunk 1]
```diff
@@ -42,7 +43,9 @@
 
     'typogrify',
     'social_auth',
+    'adminsortable2',
     'django_gravatar',
+    'django_extensions',
```
Comment (bahattincinic) [inline web/main/settings.py]: Why do you need `django-extensions` ? I remembered that we tried to use this package. but we did not use it. maybe we forgot to remove it in requirements file.

Comment (bahattincinic) [reply web/main/settings.py]: Maybe we can use in the local environment.

Comment (KrzysztofMadejski) [reply web/main/settings.py]: It has some helpful features that can be used in development like `python manage.py show_urls`. Doesn't really relate to this issue, but would be helpful to have it in the future.

File: web/nouns/models.py [hunk 2]
```diff
@@ -209,12 +209,16 @@ def relation_type_label(self):
...
     title = models.CharField(max_length=255)
     slug = models.CharField(max_length=255)
     nouns = models.ManyToManyField('Noun', null=True, blank=True)
-    order = models.IntegerField()
-    language = models.CharField(max_length=255, blank=True, null=True)
-    is_featured = models.BooleanField(max_length=255, default=False)
+    order = models.IntegerField(default=0)
+    language = models.CharField(max_length=255, choices=settings.LANGUAGES, default=get_language)
+    # Translators: this is a Channel's is_featured field
+    is_featured = models.BooleanField(max_length=255, default=False, help_text=_('If set this channel will be shown on the main page'))
```
Comment (bahattincinic) [inline web/nouns/models.py]: This change needs Django migrations. If any model property changed (DB changes or not), User should generate Django Migration.

Can you add Django migrations about it ?

Comment (KrzysztofMadejski) [reply web/nouns/models.py]: I've added migrations for nouns. There were some other missing premises migrations though (e8fd225d0f35f680d8a9) . I've added them as a #366 PR and also included here. If they look ok, just merging this PR will solve the other as well.

Comment (KrzysztofMadejski) [reply web/nouns/models.py]: BTW: It's weird that non-db changes are tracked in migrations as well. It has been resolved in Django 1.9.1 apparently, here is a hotfix to disable it: https://stackoverflow.com/questions/26503826/why-does-django-generate-migrations-for-help-text-and-verbose-name-changes

File: web/main/settings.py (hunk 1 above)
Comment (bahattincinic) [reply web/main/settings.py]: I agree with you. we can use it .

**Comment by bahattincinic** [comment]
thanks - it seems okay in the first glance.

I need 1 week to get this in though.

**Pull request closed without merging**